src/mozilla_sec_eia/models/sec10k/__init__.py

```diff
@@ -28,13 +28,14 @@
     mlflow_train_test_io_managers,
 )
 
-from . import basic_10k, ex_21, extract
+from . import basic_10k, ex_21, extract, sec_output_table
```

Comment:
I modified this module to mimic what you did with the Basic 10k and Ex. 21 assets for the new SEC output table asset.